ledger-api/grpc-definitions/com/daml/ledger/api/v1/active_contracts_service.proto

```diff
@@ -45,6 +45,10 @@ message GetActiveContractsRequest {
   // If not set the current ledger end offset will be used.
   // Optional
   string active_at_offset = 4;
+
+  // Must be a valid domain ID
```

Comment:
Should we add some kind of annotation to track updates that need to be done once we split 2.y and 3.y ? (such as making domain ID mandatory)

Reply:
I think that would be the 3.0 PR no? (https://github.com/digital-asset/daml/pull/15471)
From my part I would not confuse readers with hints on what will change and possible how later. WDYT?

Reply:
It would be both for readers but also for us: to have a list of things that need to be revisited. It can also be tracked in an issue if you think it is better

Reply:
I started placing comments everywhere where 3.0 changes will come.